src/styles/ant-vars.less

```diff
@@ -142,14 +142,14 @@ https://github.com/ant-design/ant-design/blob/master/components/style/themes/def
 @checkbox-group-item-margin-right: 8px;
 
 // Radio buttons
-@radio-button-bg: @black;
-@radio-button-checked-bg: @dark-three;
-@radio-button-color: @grayish-brown;
-@radio-button-hover-color: @primary-5;
-@radio-button-active-color: @dark-three;
-@radio-disabled-button-checked-bg: tint(@black, 90%);
-@radio-disabled-button-checked-color: @disabled-color;
-@radio-wrapper-margin-right: 8px;
+// @radio-button-bg: @black;
+// @radio-button-checked-bg: @dark-three;
+// @radio-button-color: @grayish-brown;
+// @radio-button-hover-color: @primary-5;
+// @radio-button-active-color: @dark-three;
+// @radio-disabled-button-checked-bg: tint(@black, 90%);
+// @radio-disabled-button-checked-color: @disabled-color;
+// @radio-wrapper-margin-right: 8px;
```

Comment:
why are these commented out? if w're not going to use them they should just be deleted. but it does look like we had set these up at some point, are there radio buttons somewhere in the app that are using these?

Reply:
So I couldn't find anywhere that they seemed to be used, but there are lots of parts of this file that are commented out rather than deleted when un-used... I wasn't sure if we were trying to retain some past config for use when switching to antd 5/away from less vars?

Reply:
https://github.com/simularium/simularium-website/pull/550